arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -338,7 +331,7 @@
         "tags": [
           "WorkspaceCollections"
         ],
-        "operationId": "WorkspaceCollections_listBySubscription",
+        "operationId": "WorkspaceCollections_list",
```

Comment:
isn't listBySubscription or listBySubscriptionID more appropriate in this case? It also matches the description.